di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/utils/HandlerHelper.java

```diff
@@ -488,6 +494,16 @@ public boolean verifySignedJwt(SignedJWT signedJWT, CredentialIssuer credentialI
                                         credentialIssuer.publicVCSigningVerificationJwkBase64()))));
     }
 
+    private String getHashedKeyId(String keyId) {
+        try {
+            MessageDigest digest = MessageDigest.getInstance("SHA-256");
+            byte[] hash = digest.digest(keyId.getBytes(StandardCharsets.UTF_8));
+            return Hex.encodeHexString(hash);
```

Comment:
Do we need a whole library added just to do this? 

```
public static String bytesToHex(byte[] bytes) {
    StringBuilder hex = new StringBuilder();
    for (byte b : bytes) {
        hex.append(String.format("%02x", b));
    }
    return hex.toString();
}
```

Reply:
Maybe not, I didn't really think about it this what spot have done and what's in the RFC